File: phyton/Clothes.py
from Accessories import Accessories

class Clothes(Accessories):
    # konstruktor
    def __init__(self, id, name, price, stock, type, material, color, category, size, brand):
        super().__init__(id, name, price, stock, type, material, color)
        self.category = category
        self.size = size
        self.brand = brand
        
    # getter dan setter untuk category
    def setCategory(self, category):
        self.category = category
    
    def getCategory(self):
        return self.category
    
    # getter dan setter untuk size
    def setSize(self, size):
        self.size = size
    
    def getSize(self):
        return self.size
    
    # getter dan setter untuk brand
    def setBrand(self, brand):
        self.brand = brand
    
    def getBrand(self):
        return self.brand

File: cpp/Clothes.cpp
#include <bits/stdc++.h>
#include "Accessories.cpp"

using namespace std;

class Clothes : public Accessories
{
private:
    string category;
    string size;
    string brand;

public:
    // konstruktor kosong
    Clothes()
    {
    }

    // konstruktor dengan parameter untuk mengisi atribut
    Clothes(string category, string size, string brand)
    {
        this->category = category;
        this->size = size;
        this->brand = brand;
    }

    // setter category
    void setCategory(string category)
    {
        this->category = category;
    }

    // getter category
    string getCategory()
    {
        return category;
    }

    // setter size
    void setSize(string size)
    {
        this->size = size;
    }

    // getter size
    string getSize()
    {
        return size;
    }

    // setter brand
    void setBrand(string brand)
    {
        this->brand = brand;
    }

    // getter brand
    string getBrand()
    {
        return brand;
    }

    ~Clothes()
    {
    }
};
